src/Symfony/Component/Validator/Tests/Constraints/ExpressionValidatorTest.php

```diff
@@ -235,4 +235,17 @@ public function testExpressionLanguageUsage()
 
         $this->assertTrue($used, 'Failed asserting that custom ExpressionLanguage instance is used.');
     }
+
+    public function testExpressionLanguageUsageWithCustomDataPath()
```

Comment:
Yes I saw that. But the previous test actually tests that passing in a custom `ExpressionLanguage` instance works. Hence the name :smile: